nielslange/woo-test-environment
# Install stripe plugin and configure test env keys

Fixes https://github.com/nielslange/woo-test-environment/issues/5

This PR allows installing and configuring the WooCommerce Payment Gateway plugin by asking the user for the publishable and secret key for the Stripe test environment.

## Testing steps

1. Check out this repo.
2. Switch to the `add/5-setup-stripe` branch.
3. Install the WP-CLI custom command using:
```
wp package install <path-to-your-local-repo>
```
4. Setup a new site with:
```
wp woo-test-environment setup
```
5. Enter the `publishable` and `secret` Stripe keys when you are prompted by the command.
6. Go to `/wp-admin/admin.php?page=wc-settings&tab=checkout&section=stripe&panel=settings` on your site.
7. Check the `Enable Stripe` and `Enable test mode` checkboxes are checked.
8. Click on `Edit account keys`, check that the keys you entered are the same that appear on the `Test` settings.
